source/pci-dss/configuration-assessment.rst

```diff
@@ -1,33 +1,33 @@
 
-To achieve the above requirements, SCA can run checks to assess whether it is necessary to change password related configuration to ensure strong passwords, remove unnecessary software, disable unnecessary services, or audit the TCP/IP stack configuration. Sources of industry-accepted system hardening standards may include, but are not limited to: Center for Internet Security (CIS), International Organization for Standardization (ISO), SysAdmin Audit Network Security (SANS), National Institute of Standards Technology (NIST).
+To achieve the above requirements, SCA runs assessment checks. These checks assess whether it is necessary to change password related configuration to ensure strong passwords, remove unnecessary software, disable unnecessary services, or audit the TCP/IP stack configuration. Sources of system hardening standards accepted by the industry include, but are not limited to: the Center for Internet Security (CIS), the International Organization for Standardization (ISO), SysAdmin Audit Network Security (SANS), National Institute of Standards Technology (NIST).
 
-Out-of-the-box, Wazuh includes CIS baselines for a wide range of operating systems and applications. These operating systems include Debian, macOS, Red hat, Windows, among others. For more information, see a :ref:`list of the available SCA policies <share_policy_files_and_configuration_with_the_Wazuh_agents>`. Other baselines can be created for other systems or applications as well. More details on configuring SCA checks can be found in the :doc:`SCA documentation section </user-manual/capabilities/sec-config-assessment/index>`.
+Out-of-the-box, Wazuh includes CIS baselines for a wide range of operating systems and applications. This includes Debian, macOS, Red hat, and Windows operating systems. For more information, see a :ref:`list of the available SCA policies <share_policy_files_and_configuration_with_the_Wazuh_agents>`. You can create other baselines for other systems or applications as well. Find more details on configuring SCA checks in the :doc:`SCA documentation section </user-manual/capabilities/sec-config-assessment/index>`.
 
 
 Use cases
 ---------
 
 Below are some PCI DSS requirement use cases that can be met with the SCA module.
 
-- PCI DSS 2.2.4 requires that only necessary services, protocols, daemons, and functions are enabled, and all unnecessary functionality is removed or disabled. An example of a system service that may be abused if it is misconfigured is IP forwarding. When IP forwarding is configured on a device, it may serve as a router and can be abused.
+- PCI DSS 2.2.4 requires that only necessary services, protocols, daemons, and functions are enabled, and all unnecessary functionality is removed or disabled. An example of a system service that may be abused if it is misconfigured is IP forwarding. When IP forwarding is configured on a device, it may serve as a router to be abused.
```

Comment:
I suggest this 3rd option:
PCI DSS 2.2.4 requires enabling only necessary services, protocols, daemons, and functions and removing or disabling all unnecessary functionality. IP forwarding is an example of a system service that may be abused if misconfigured. When IP forwarding is configured on a device, it may serve as a router to be abused.